packages/web-client/tests/acceptance/issue-prepaid-card-test.ts

```diff
@@ -14,18 +14,18 @@ import { toWei } from 'web3-utils';
 import BN from 'bn.js';
 
 import { DepotSafe } from '@cardstack/cardpay-sdk/sdk/safes';
-import { encodeDID } from '@cardstack/did-resolver';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 import prepaidCardColorSchemes from '../../mirage/fixture-data/prepaid-card-color-schemes';
 import prepaidCardPatterns from '../../mirage/fixture-data/prepaid-card-patterns';
-import { Response as MirageResponse } from 'ember-cli-mirage';
 import { timeout } from 'ember-concurrency';
 import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
 import { faceValueOptions } from '@cardstack/web-client/components/card-pay/issue-prepaid-card-workflow/workflow-config';
 
 import { MirageTestContext } from 'ember-cli-mirage/test-support';
 
-interface Context extends MirageTestContext {}
+interface Context extends MirageTestContext {
+  normalizedRequestAttrs(): any;
```

Comment:
I haven't needed this property in local tests, does anything here use normalizedRequestAttrs?

Nice to know that this convenience method exists though!

Reply:
ah that’s an autofix that I neglected to remove after moving the handling into `mirage/config.js`, thanks!